scripts/eslint-rules/warning-and-invariant-args.js

```diff
@@ -79,6 +89,21 @@ module.exports = function(context) {
           }
         );
       }
+
+      if (node.callee.name === 'invariant') {
+        if (!messages.has(format)) {
+          context.report(
+            node,
+            'Error message does not have a corresponding production ' +
+              'error code.\n\n' +
+              'Run `yarn extract-errors` to add the message to error code ' +
+              'map, so it can be stripped from the production builds. ' +
+              "Alternatively, if you're updating an existing error " +
+              'message, you can modify ' +
+              '`scripts/error-codes/codes.json` directly.'
```

Comment:
Seems like it would be nice to log the error message too.

Reply:
I figured this was fine because if you have ESLint in your editor, it will display the lint error inline with the message.